tests/Microsoft.Bot.Schema.Tests/Teams/MessageActionsPayloadTests.cs

```diff
@@ -0,0 +1,388 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Bot.Schema.Teams;
+using Xunit;
+
+namespace Microsoft.Bot.Schema.Tests.Teams
+{
+    /// <summary>
+    /// Tests to ensure that MessageActionsPayload works as expected.
+    /// </summary>
+    public class MessageActionsPayloadTests
+    {
+        /// <summary>
+        /// Ensures the constructor of the <see cref="MessageActionsPayload"/> class works as expected.
+        /// </summary>
+        [Fact]
+        public void TestMessageActionsPayloadConstructor()
+        {
+            var messageActionPayload = new MessageActionsPayload();
+
+            Assert.NotNull(messageActionPayload);
```

Comment:
We should assert the defaults are set here, otherwise the test doesn't make a lot of sense.

Reply:
Feedback applied. Thanks!